src/pages/billing/NewBillingProjectWizard/StepWizard/Step.ts

```diff
@@ -0,0 +1,37 @@
+import { CSSProperties } from 'react'
+import { section } from 'react-hyperscript-helpers'
+import colors from 'src/libs/colors'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+function stepBanner(active: boolean): CSSProperties {
+  return {
+    borderRadius: '8px',
+    boxSizing: 'border-box',
+    padding: '1.5rem 2rem',
+    marginTop: '1rem',
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    border: active ? `1px solid ${colors.accent()}` : `1px solid ${colors.accent(0.2)}`,
+    backgroundColor: active ? colors.light(0.5) : colors.light(0.3),
+    boxShadow: active ? '0 0 5px 0 rgba(77,114,170,0.5)' : 'none'
```

Comment:
This shadow color was hard-coded before? Just wondering if it will look OK with other whitelabel themes.

Reply:
yes - I generally try *very* hard to avoid hardcoding values like that. I don't think I touched the colors after moving this from the original file.

I just plugged it into a converter. and the hex value came out to `#4d72aa80`
The 'accent' color value is `#4d72aa`, which translates to `rgba(77, 114, 170, 1)`
So should this be `colors.accent(.5)`?

Reply:
I just tested it out, and it looks correct / the same - so I'm going with it.

Reply:
Sounds good.